app-text/mupdf/files/mupdf-1.20.0-lcms2.patch

```diff
@@ -0,0 +1,40 @@
+From 9f3bd926a74292adfff78186b184b4b494babd4e Mon Sep 17 00:00:00 2001
+From: =?UTF-8?q?Philipp=20R=C3=B6sner?= <rndxelement@protonmail.com>
+Date: Fri, 1 Jul 2022 22:35:20 +0200
+Subject: [PATCH] Fix build errors with -Wstrict-aliasing
+
+Building with the -Wstrict-aliasing C-flag resulted in build errors
+in src/cmsplugin.c.
+Using memcpy resolves this, as it is a correct way for
+type punning in C.
+
+Part of a fix for issue https://github.com/mm2/Little-CMS/issues/303
+See PR: https://github.com/mm2/Little-CMS/pull/323
+--- a/thirdparty/lcms2/src/cmsplugin.c
++++ b/thirdparty/lcms2/src/cmsplugin.c
+@@ -177,7 +177,11 @@ cmsBool CMSEXPORT  _cmsReadFloat32Number(cmsContext ContextID, cmsIOHANDLER* io,
+     if (n != NULL) {
+ 
+         tmp = _cmsAdjustEndianess32(tmp);
+-        *n = *(cmsFloat32Number*)(void*)&tmp;
++
++        #if __STDC_VERSION__ >= 201112L
++        static_assert(sizeof(*n) == sizeof(tmp), "*n and tmp have different sizes");
```

Comment:
does the file contain the right include already?

Reply:
Yes, the file includes `lcms2_internal.h`, which includes `assert.h`

Reply:
I've replied on the upstream PR, let's hope we can get it in with a `union` instead of the `memcpy`.